src/yaws_api.erl

```diff
@@ -891,6 +891,7 @@ url_decode_q_split([$%, Hi, Lo | Tail], Ack) ->
        %% RFC 3986 section 2.2 says that encoded reserved characters
        %% should not be decoded, otherwise the meaning of the URL data
        %% changes
+       Hex == $\r; Hex == $\n; %% prevent CRLF injection
```

Comment:
Note the comment preceding this line. Since these characters are not part of the portion of RFC 3986 that the rest of the characters here come from, please move this line above the comment.

Reply:
Sorry, you're right.